src/components/PdButton.css

```
@@ -0,0 +1,168 @@
+  color: var(--color-primary);
+}
+
+.pd-button.pd-button--raised .pd-button__background svg {
+  overflow: visible;
+  stroke: transparent;
+  stroke-width: 0;
+  fill: currentColor;
+}
+
+.pd-button.pd-button--raised .pd-button__background svg path {
+  filter: url(#pd-shadow);
```

Stylelint (17.3.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)